src/client/utils/dom/dom.ts

```diff
@@ -165,3 +165,11 @@ export function classNames(...args: Array<string | Record<string, any>>): string
 
   return classes.join(" ");
 }
+
+export function originatesFromTextAreaOrInput(e: KeyboardEvent): boolean {
+  const element = e.target as HTMLElement;
+  if (element.tagName === "INPUT" || element.tagName === "TEXTAREA") {
```

Comment:
This could just be `return element.tagName === "INPUT" || element.tagName === "TEXTAREA"`

Reply:
That should have occurred to me 😅